package.json

```diff
@@ -5,10 +5,12 @@
   "productName": "Activist Handbook Editor",
   "author": "Activist Handbook",
   "private": true,
+  "workspaces": ["firebaseFunctions"],
   "scripts": {
-    "👩‍💻dev": "quasar dev",
-    "🔥deploy": "firebase deploy",
-    "🔒rules": "firebase deploy --only firestore:rules",
+    "dev": "quasar dev",
+    "deploy": "firebase deploy",
+    "rules": "firebase deploy --only firestore:rules",
+    "emu": "firebase emulators:start",
```

Comment:
Can you actually try it running yourself? eg. download a whole new repo, switch to this branch, and run `yarn emu` in the root, and then `yarn dev`?

Reply:
hmm, although now that i think about it, you have win10 if I am not mistaken. I hope it won't be too different :sweat_smile: